packages/grid/x-data-grid-pro/src/tests/statePersistence.DataGridPro.test.tsx

```diff
@@ -128,6 +151,57 @@ describe('<DataGridPro /> - State Persistence', () => {
       expect(apiRef.current.exportState()).to.deep.equal(FULL_INITIAL_STATE);
     });
 
+    it('should export the controlled values of the models', () => {
+      render(
+        <TestCase
+          filterModel={FULL_INITIAL_STATE.filter?.filterModel}
+          sortModel={FULL_INITIAL_STATE.sorting?.sortModel}
+          columnVisibilityModel={FULL_INITIAL_STATE.columns?.columnVisibilityModel}
+          page={FULL_INITIAL_STATE.pagination?.page}
+          pageSize={FULL_INITIAL_STATE.pagination?.pageSize}
+          pinnedColumns={FULL_INITIAL_STATE.pinnedColumns}
+          // Some portable states don't have a controllable model
+          initialState={{
+            columns: {
+              orderedFields: FULL_INITIAL_STATE.columns?.orderedFields,
+              dimensions: FULL_INITIAL_STATE.columns?.dimensions,
+            },
+            preferencePanel: FULL_INITIAL_STATE.preferencePanel,
+          }}
+        />,
+      );
+      expect(apiRef.current.exportState()).to.deep.equal(FULL_INITIAL_STATE);
+    });
+
+    it('should export the controlled values of the models when using exportOnlyUsedModels', () => {
```

Comment:
`exportOnlyUsedModels` is not used here. Was that the intention?

```suggestion
    it('should export only the controlled values of the models when not using exportOnlyUsedModels', () => {
```

Reply:
The goal was to check that if we pass some controlled models, they are correcftly exported
I'm not puttin "only" because I'm passing all possible controlled model (to test that all of them are exported).
But yes I forgot to pass `exportOnlyUsedModels`, fixed